src/ngraph/runtime/cpu/cpu_backend.cpp

```diff
@@ -50,6 +51,23 @@ extern "C" runtime::BackendConstructor* get_backend_constructor_pointer()
     return s_backend_constructor.get();
 }
 
+#ifndef CPU_BACKEND_STATIC
+extern "C" runtime::BackendConstructor* get_backend_constructor_pointer()
+{
+    return runtime::cpu::get_backend_constructor_pointer();
+}
+#endif
+
+void runtime::cpu::static_initialize()
```

Comment:
Comment here about what this does and where it is called from ? How it is related to building static lib ?